ci-scripts/release-tests.sh

```diff
@@ -26,6 +30,7 @@ export SCALE_DB_STORAGES='$SCALE_DB_STORAGES'
 export SCALE_MEMORY_REQUESTS_LIMITS=:
 export SCALE_REPLICAS=1
 export SCALE_WORKERS=20
+export USERS=100
```

Comment:
@jhutar, It is OK to use `mvp-scalability` to run a single run of the MVP test, you just need tu utilize all the `SCALE_*` variables instead of the sinlge-value ones. For instance, for `USERS` it is sufficient to use the `SCALE_ACTIVE_USERS_SPAWN_RATES` and the `test-scalability.sh` script will set the `USERS` for that single run based on that. Additionally, you should also enable RBAC
```suggestion
export ENABLE_RBAC=true
```